ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/validator/batch/ConflictDetection.scala

```diff
@@ -104,8 +105,7 @@ class ConflictDetection(val damlMetrics: Metrics) {
           // NOTE(JM): We show the template id as the other piece of data we have is the contract key
           // hash, which isn't very useful as it's an implementation detail not exposed over ledger-api.
           val templateId =
-            ValueCoder
-              .decodeIdentifier(key.getContractKey.getTemplateId)
+            Try(Conversions.decodeIdentifier(key.getContractKey.getTemplateId))
```

Comment:
Do we want to just swallow this exception?

Reply:
Yes, this was the case previously as well, as this is just for logging.